experiments/notebooks/visualizations/eth_supply_simulator.py

```diff
@@ -115,7 +115,7 @@
                         75: '75',
                         100: '100'
                     },
-                    value=25,
+                    value=30,
```

Comment:
Can we set these defaults values at the top of the file, and pass in to both the variable and the annotations? Will make updates more seamless in future.

Reply:
Sure, we can add a variable for `default_basefee`, and the same for the other dropdowns